src/DocumentDB.ChangeFeedProcessor.IntegrationTests/IntegrationTest.cs

```diff
@@ -160,7 +164,7 @@ private async Task CreateMonitoredCollectionAsync(string monitoredCollectionName
 
             if (this.IsPartitionedMonitoredCollection)
             {
-                monitoredCollection.PartitionKey = new PartitionKeyDefinition { Paths = { "/id" } };
+                monitoredCollection.PartitionKey = new PartitionKeyDefinition { Paths = { "/partitionId" } };
```

Comment:
why do we need to change this?

Reply:
Should this be conditional? `/id` is fine, maybe you want to have 1 test that changes it to something else using the new config

Reply:
We should have a test for typical case, i.e. PKDef = /id. Can we keep old way, plus add a new test that's using /partitionId?

Reply:
made it conditional similar to leaseCollection like Matias suggested earlier